changelog.d/pr-7470

```diff
@@ -0,0 +1,12 @@
+synopsis: Suppress warnings for verbosity
+packages: cabal-install
+prs: #7470
+issues: #7286
+
+description: {
+- adds a verbosity flag "+nowarn", to suppress all warnings
+- adds a verbosity flag "+nowarnexperimental", to suppress 
+  all warnings from experimental features
+- removes the warnings for extraneous versions
+  https://github.com/haskell/cabal/pull/7470#issuecomment-875878507
+}
```

Comment:
oh, and technically this should be two separate entries

Reply:
huh, how would that work? I thought it was only one changelog entry per PR.

Reply:
> I thought it was only one changelog entry per PR

CONTRIBUTING.md is indeed unclear

#7486 :)

Reply:
Done.